go.mod

```diff
@@ -5,19 +5,19 @@ go 1.14
 require (
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/pkg/errors v0.9.1
-	github.com/spf13/cobra v0.0.6
+	github.com/spf13/cobra v1.0.0
```

Comment:
@rhuss : I'll raise another PR to pin cobra at master with entry for `replace` section removed.

Reply:
👍